ios/listdevices.go

```diff
@@ -102,9 +103,47 @@ func (muxConn *UsbMuxConnection) ListDevices() (DeviceList, error) {
 	if err != nil {
 		return DeviceList{}, fmt.Errorf("Failed getting devicelist: %v", err)
 	}
+	muxResponse := MuxResponsefromBytes(response.Payload)
+	if muxResponse.Number == 4294967295 {
```

Comment:
it looks like a magic number when put into decimal. maybe use 0xffffffff or a constant?
also, why are we getting this number instead of a device list? can you elaborate please?

Reply:
This number should be -1, Bu the Number Type is uint32. So it is 0xffffffff;